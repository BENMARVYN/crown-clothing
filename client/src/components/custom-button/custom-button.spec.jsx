import React from "react";
import { shallow } from "enzyme";
import CustomButton from "./custom-button.component";

describe("Testing CustomButton", () => {
  it("Should match snapshot", () => {
    expect(shallow(<CustomButton children="Test" />)).toMatchSnapshot();
  });
});
