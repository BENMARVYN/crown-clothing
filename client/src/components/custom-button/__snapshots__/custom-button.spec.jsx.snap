// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing CustomButton Should match snapshot 1`] = `ShallowWrapper {}`;
